src/filesys.cpp

```diff
@@ -740,5 +740,26 @@ bool Rename(const std::string &from, const std::string &to)
 	return rename(from.c_str(), to.c_str()) == 0;
 }
 
+bool bad_char(const char c){
+	if (c >= 0 && c <= 31)
+		return true;
+	if (bad_chars.find_first_of(c) != std::string::npos)
+		return true;
+	if (c > 255)
+		return true;
+	return false;
+}
+
+void FixBadFilename(std::string &input){
```

Comment:
@rubenwardy no interest to return std::string, just pass std::string & and return void